content/_data/changelogs/weekly.yml

```diff
@@ -298,6 +298,8 @@
     - type: major rfe
       message: >
         <strong>Jenkins now requires Java 8 to run.</strong>
+        This applies to JREs running all Jenkins components including master, agents, CLI and related components like Maven Interceptors in Maven Project Plugin. (Note that JDKs for builds are configured independently of the <code>java</code> running the Jenkins processes)
```

Comment:
I am not sure the brackets are correct for all job types, and I would rather move details to the linked blog post